### PR TITLE
Patch 5 - corrections in chapter cue lists

### PR DESCRIPTION
**Pull request by icke-siegen**
various reformatting and corrections in cuelists, see remarks per commit

### CODE REVIEW

File: docs/cue-lists/theatre-programming.md [hunk 1]
```diff
@@ -112,72 +66,24 @@ recorded)*. To record everything with intensity above zero, switch
...
----
-
-````xml
-<Copy> <Cue> n <@> <Enter>
-````
-
-Copy cue **n** onto the end of the cue list
+Keypresses | Result
+---- | ----
+\<Record\> {blue fader button} {blue fader button} | Record the next cue onto the end of the cue list on this fader
```
Comment (garethnunns) [inline docs/cue-lists/theatre-programming.md:71]: We refer to this in a few different ways in the manual:

- "blue fader button"
- "blue button"
- Swop
- swop

We should try be consistent (I appreciate this is the theatre programming page)

Comment (icke-siegen) [reply docs/cue-lists/theatre-programming.md:71]: Add the notorious "select", "Select" and the occasional "handle button". 100% d'accord, but I don't have a solution.

Maybe a very short section on this - maybe in conjunction with key profiles. And then, if possible, a special notation which is parsed into a formatted thingy like \< ... \> for hard keys and \[ ... \] for softkeys/menubuttons. Maybe we could use \{ curly braces \} and have this rendered with blue background? 

As for the name, maybe 'Select/Swop' would be the best solution

**Comment by congoblue** [comment]
The blue buttons have changed names many times through the different console designs. Swop was the name from the old Rolacue consoles, from which this manual text has evolved.

I don't think any front panel prints call them "Swop" any more so that would only be of use to longtime Avolites users who would probably understand anyway.

I think consistently calling them "Select" buttons makes the most sense, as that is nearly always the function, (except when you are using them in Swop mode to run a show). But they are not labelled as such on any console so we should have a section somewhere near the beginning explaining which buttons we are referring to.

**Comment by congoblue** [comment]
Also, this section includes a lot of softkey letters "A" "B" etc, I have tried to remove these throughout the manual as they sometimes change with different software versions, so just show the softkey text /[like this/]. Obviously never did this section.

File: docs/cue-lists/theatre-programming.md (hunk 1 above)
Comment (garethnunns) [reply docs/cue-lists/theatre-programming.md:71]: Ok we'll shelve this conversation and maybe make it an issue that can be resolved across the manual

**Review by garethnunns — APPROVED** [review]

File: docs/cue-lists/theatre-programming.md (hunk 1 above)
Comment (garethnunns) [reply docs/cue-lists/theatre-programming.md:71]: I've open #189 to continue this discussion

**Pull request merged**